docs/cluster_spec.md

```diff
@@ -377,6 +377,8 @@ Specifying KubeDNS will install kube-dns as the default service discovery.
 
 This will install [CoreDNS](https://coredns.io/) instead of kube-dns.
 
+**Note:** If you are upgrading to CoreDNS, kube-dns will be left in place and must be removed manually. You can scale the kube-dns and kube-dns-autoscaler deployments in the `kube-system` namespace to 0 as a starting point
```

Comment:
Mind adding a period at the end of the sentence? 

Do you think we should add a note about _not_ deleting the `kube-dns` service?

Reply:
Good call, yes I will update to not delete the service and note why!